server/sonar-server/src/main/resources/org/sonar/server/component/ws/show-example.json

```diff
@@ -9,13 +10,15 @@
   },
   "ancestors": [
     {
+      "organization": "my-org-1",
```

Comment:
As ancestors are in the same organization as "component", I wonder if this field deserves to be returned.